src/app/static/src/app/store/stepper/getters.ts

```diff
@@ -20,5 +22,12 @@ export const getters: StepperGetters & GetterTree<StepperState, RootState> = {
             5: rootGetters['modelRun/complete'],
             6: false
         }
+    },
+    laterCompleteSteps: (state: StepperState, getters: any) => {
```

Comment:
I suppose a limitation of this is that it won't prevent you from clearing a later partially completed step. Not sure there's an obvious way to prevent this. Could mark a state as partially complete when it gets saved to local storage and isn't identical to its initial state? Might not be worth it. But user might be surprised at being warned about some changes being lost but not others..

Reply:
Yeah I did consider this. We could have a property "edited" or something like that, but overall I think not worth it. The only step this applies to currently is the model options - if you have filled out some model options but not validated them, they will be reset without warning. But I think this is a minor inconvenience.